tools/sync_test_server/ros-testing-server.js

```diff
@@ -66,6 +66,7 @@ function startRealmObjectServer(onSuccess, onError) {
             syncServerChildProcess = spawn('ros',
                     ['start',
                         '--data', path,
+			'--loglevel', 'all',	
```

Comment:
Do we want to merge this? Doesn't it make looking at the logs rather hard in general?

Reply:
I'll remove it, this is only helpful locally since AFAIK we don't capture the ROS log (within Docker) from CI. we'll revert